src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable jsx-quotes */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Please, consider removing all linter disables - let the linter makes its direct job, aka help you find bugs in the early stages of development 👍